api/src/services/listing.service.ts

```diff
@@ -283,36 +283,36 @@ export class ListingService implements OnModuleInit {
   ): Promise<{ [key: string]: string[] }> {
     const userResults = await this.prisma.applications.findMany({
       select: {
-        language: true,
```

Comment:
This is not related to these specific changes, but I'm curious why this function is in the listing service? It's only being used by the lottery service and it never touches the listing db table. Would this make more sense in either the lottery or application services?

Reply:
Good q I'm not super sure! My guess is because it's related to the function above `getUserEmailInfo`

Reply:
Moved it into lottery 👍